<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnK1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Key1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnK2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Key2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnK3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Key3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnK4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Key4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnK5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Key5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnK6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Key6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnK7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Key7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnK8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Key8</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
